### DISCUSSION TITLE
ydotool for Keypress

### DISCUSSION THREAD

**Discussion by abhijeet-singh800** [bulletmark/libinput-gestures#365] [Announcements]
As we know that `xdotool` only works for x11 there is a project called [ydotool](https://github.com/ReimuNotMoe/ydotool) that achieves the same thing a little differently but works on Wayland too if given the go-ahead I would like to update the Readme  file in coming weeks (I'm stretched a little thin with collage exams) with this tool as an alternative for the `xdotool`

**Comment by bulletmark** [comment]
We have discussed ydotool [plenty of times](https://github.com/bulletmark/libinput-gestures/issues?q=is%3Aissue+ydotool) around here previously. The documentation and implementation is poor. It is written in C but you will notice the author is just about to [rewrite the entire application in Javascript](https://github.com/ReimuNotMoe/ydotool?tab=readme-ov-file#2024-roadmap) so, no I am not interested in it, certainly at present. Let's see where it gets eventually.
